qa/L0_onnxtrt_optimization/test.sh

```diff
@@ -120,6 +130,12 @@ for MODEL in \
         RET=1
     fi
 
+    grep "TensorRT Execution Accelerator is set for '${MODEL}_cache_on'" $SERVER_LOG
+    if [ $? -ne 0 ]; then
+        echo -e "\n***\n*** Failed. Expected TensorRT Execution Accelerator is set\n***"
```

Comment:
This message should say something about cache one, perhaps similar to the message 2 above